src/Views/Form/Templates/Sequoia/Sequoia.php

```diff
@@ -57,21 +57,32 @@ public function loadHooks() {
 
 	/**
 	 * @inheritDoc
+	 * @unreleased Load google fonts if "enabled".
 	 */
 	public function loadScripts() {
 
 		// Localize Template options
 		$templateOptions = FormTemplateUtils::getOptions();
 
 		// Set defaults
+		$templateOptions['style']['google-fonts']                 = ! empty( $templateOptions['style']['google-fonts'] ) ? $templateOptions['style']['google-fonts'] : "enabled";
 		$templateOptions['introduction']['donate_label']          = ! empty( $templateOptions['introduction']['donate_label'] ) ? $templateOptions['introduction']['donate_label'] : __( 'Donate Now', 'give' );
 		$templateOptions['introduction']['primary_color']         = ! empty( $templateOptions['introduction']['primary_color'] ) ? $templateOptions['introduction']['primary_color'] : '#28C77B';
 		$templateOptions['payment_amount']['next_label']          = ! empty( $templateOptions['payment_amount']['next_label'] ) ? $templateOptions['payment_amount']['next_label'] : __( 'Continue', 'give' );
 		$templateOptions['payment_amount']['header_label']        = ! empty( $templateOptions['payment_amount']['header_label'] ) ? $templateOptions['payment_amount']['header_label'] : __( 'Choose Amount', 'give' );
 		$templateOptions['payment_information']['header_label']   = ! empty( $templateOptions['payment_information']['header_label'] ) ? $templateOptions['payment_information']['header_label'] : __( 'Add Your Information', 'give' );
 		$templateOptions['payment_information']['checkout_label'] = ! empty( $templateOptions['payment_information']['checkout_label'] ) ? $templateOptions['payment_information']['checkout_label'] : __( 'Process Donation', 'give' );
 
-		wp_enqueue_style( 'give-google-font-montserrat', 'https://fonts.googleapis.com/css?family=Montserrat:100,100i,200,200i,300,300i,400,400i,500,500i,600,600i,700,700i,800,800i,900,900i&display=swap', [], GIVE_VERSION );
+		$isGoogleFontEnabled = give_is_setting_enabled( $templateOptions['style']['google-fonts'] );
+
+		if ( $isGoogleFontEnabled ) {
+			wp_enqueue_style(
+				'give-google-font-montserrat',
+				'https://fonts.googleapis.com/css?family=Montserrat:100,100i,200,200i,300,300i,400,400i,500,500i,600,600i,700,700i,800,800i,900,900i&display=swap',
```

Comment:
Do we use all of these font weights and styles? This is just a lot of data (close to 2MB if I’m not wrong). It might be good to optimize this so that the user doesn’t have to do it themselves.

Reply:
@knowler I search code and we utilize only `400`, `500`, `600`, `900`, normal aka `100`.  I need you help on this one. Can you also search code and let me know if we want to include same: `src/Views/Form/Templates/Sequoia/assets`

Reply:
What is "normal"? 400 is CSS `font-weight: normal`.

Reply:
@knowler I updated my comment. I miss understood `normal` font weigth.